mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -31,3 +48,7 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 	METHOD method_1343 clear ()V
 	METHOD method_16005 roundBufferSize (I)I
 		ARG 0 amount
+	METHOD method_22632 pollArrayDrawParameter ()Lcom/mojang/datafixers/util/Pair;
```

Comment:
This doesn't return just the draw parameters, but also a copy of the ByteBuffer containing the vertex data (so all the data that's needed to upload the buffer).

I think `takeData` (or `getDataAndClear`) would be much better.